### PR TITLE
refactor: remove reply Problematic field to reduce complexity

### PR DESCRIPTION
**Pull request by hi-rustin**
/hold

- [x] tests
- [x] better logs

### CODE REVIEW

**Comment by hi-rustin** [comment]
/unhold

Test on https://github.com/ti-community-infra/test-dev/pull/392 and https://github.com/ti-community-infra/test-dev/pull/393.


/assign @Mini256 

PTAL~  Thanks!

File: test/services/pull/index.test.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { SigInfoSchema } from "../../../src/config/SigInfoSchema";
 import sigInfoSchema from "../../../src/config/sig.info.schema.json";
 import { PullFormatQuery } from "../../../src/queries/PullFormatQuery";
 import { Status } from "../../../src/services/reply";
+import {migrateToJSONTip} from "../../../lib/services/messages/PullMessage";
```
Comment (Mini256) [inline test/services/pull/index.test.ts:14]: Please combine this import with line 4-8 and it should be from src.

Comment (hi-rustin) [reply test/services/pull/index.test.ts]: Good catch! Thanks for your review.

**Review by Mini256 — APPROVED** [review]

**Pull request merged**